README.md

```diff
@@ -5,7 +5,7 @@
 ## Latest Release: v2.13
 [Read the specification](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/spec.md)
 
-[Read the profile](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/profile.md)
+[Read about the platform API profile](https://github.com/openservicebrokerapi/servicebroker/blob/v2.13/profile.md)
```

Comment:
I'm not sure about using the word "platform" since its not just about platform-specific extensions. What if we just used its real name: Open Service Broker API Profile

`[Read about the Open Service Broker API Profile](...)`